### DISCUSSION TITLE
Reworking curing zombie villagers

### DISCUSSION THREAD

**Discussion by NatSavage** [ldtteam/minecolonies#8473] [Ideas]
I think the system where cured zombie villagers are turned into visitors may need a revisit. 
In my colony, I have a few vanilla villagers that I use for trading. I zombified and cured them thinking that I could make their trades cheaper but they all turned into visitors, losing me at least an hour's work to replace them.  I have two potential fixes:

- Make zombie villagers that have previously been traded with revert back to villagers. It is a cool system that you can get more colonists quickly by curing villagers however completely overwriting another existing mechanic doesn't feel right. I think this would be a good middle ground as the player likely doesn't care about villagers they have never traded with but would exclude ones that they would.

- Make curing villagers worth my while. The villagers from the cured zombies weren't even anything special. Perhaps they could be cheaper to recruit (you know because I saved their life)? higher disease immunity? work longer hours? higher base stats? just anything that would push me towards actively meaning to do this rather than just waiting for kids with only marginally lower stats.

**Comment by Kragan0** [comment]
I don't understand who thought changing this mechanic would be a good idea. The tavern is enough for gathering new colonists.
